.circleci/config.yml

```diff
@@ -599,11 +599,11 @@ workflows:
             tags:
               only: /^v[0-9]+\.[0-9]+\.[0-9]+/
             branches:
-              ignore: /.*/
+              only: dev
           requires:
             - pass-tests
           image: klaytn/klaytn
-          tag: $CIRCLE_TAG,latest
+          tag: $CIRCLE_SHA1,latest,$CIRCLE_TAG
```

Comment:
`latest` is  common for `dev` branch version? @whoisxx

Reply:
@KimKyungup Depends how to define I guess. any suggestion?

Reply:
@KimKyungup I changed to push docker image with dev tag